packages/core/helper-plugin/src/components/RelativeTime/index.js

```diff
@@ -4,7 +4,18 @@ import { intervalToDuration, isPast } from 'date-fns';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
-const RelativeTime = ({ timestamp }) => {
+/**
+ * Displays the relative time between a given timestamp and the current time.
+ * You can display a custom message for given time intervals by passing an array of custom intervals.
+ *
+ * @example <caption>Display "last hour" if the timestamp is less than an hour ago</caption>
+ * <RelativeTime
+ *  timestamp={new Date('2021-01-01')}
+ *  customIntervals={[
+ *   { unit: 'hours', threshold: 1, text: 'last hour' },
+ *  ]}
```

Comment:
```suggestion
 * @example 
 * ```jsx
 * <caption>Display "last hour" if the timestamp is less than an hour ago</caption>
 * <RelativeTime
 *  timestamp={new Date('2021-01-01')}
 *  customIntervals={[
 *   { unit: 'hours', threshold: 1, text: 'last hour' },
 *  ]}
 * ```
```